OpenOversight/app/models.py

```diff
@@ -518,6 +518,15 @@ def password(self):
     def password(self, password):
         self.password_hash = generate_password_hash(password, method="pbkdf2:sha256")
 
+    def _case_insensitive_equality(field, value):
+        return User.query.filter(func.lower(field) == func.lower(value))
+
+    def by_email(email):
+        return self._case_insensitive_equality(User.email, email)
+
+    def by_username(username):
+        return self._case_insensitive_equality(User.username, username)
```

Comment:
```suggestion
    @staticmethod
    def _case_insensitive_equality(field, value):
        return User.query.filter(func.lower(field) == func.lower(value))

    @staticmethod
    def by_email(email):
        return User._case_insensitive_equality(User.email, email)

    @staticmethod
    def by_username(username):
        return User._case_insensitive_equality(User.username, username)
```

Reply:
I'm not sure how this was working before while this wasn't a static method :thinking:

Reply:
:see_no_evil: